packages/main/src/plugin/color-registry.ts

```diff
@@ -1139,31 +1139,31 @@ export class ColorRegistry {
 
     // Podman & Kubernetes
     this.registerColor(`${status}running`, {
-      dark: colorPalette.green[400],
-      light: colorPalette.green[400],
+      dark: colorPalette.green[500],
+      light: colorPalette.green[600],
     });
     // Kubernetes only
     this.registerColor(`${status}terminated`, {
       dark: colorPalette.red[500],
-      light: colorPalette.red[500],
+      light: colorPalette.red[700],
     });
     this.registerColor(`${status}waiting`, {
       dark: colorPalette.amber[600],
       light: colorPalette.amber[600],
     });
     // Podman only
     this.registerColor(`${status}starting`, {
-      dark: colorPalette.green[600],
+      dark: colorPalette.green[500],
```

Comment:
Maybe I missed it, but starting and running is now the same colour? Should it be slightly different?

Reply:
Either way I'm 👍, as the colour difference wasn't much anyways, just wanted to confirm that this is intended.

Reply:
Yup, it is intentional: https://github.com/containers/podman-desktop/issues/6925#issuecomment-2188592434. It's the one part I'm not totally convinced about, but as you say it is a minor difference.